fetch.bs

```diff
@@ -4531,9 +4536,14 @@ Range Requests</cite>. [[HTTP-RANGE]] However, this is not widely supported by b
     <var>httpRequest</var>'s
     <a for=request>credentials mode</a>.
 
-   <li><p>Set <var>httpCache</var> to the result of <a>determining the HTTP cache partition</a>,
+   <li><p>Set <var>httpStatePartitionKey</var> to the result of <a>determining the HTTP state partition key</a>,
    given <var>httpRequest</var>.
 
+   <li><p>Set <var>httpCache</var> to null.
```

Comment:
How can it be non-null at this point?

Reply:
I assume we need to declare variables exist before conditionally setting them to something.  If this were psuedo-Javascript, it's undefined at this point, not null. If this is psuedo-C, it's a compile failure.

Anyhow, I've reverted this and added back a shortened HTTP cache partition section, per your other comment.

Reply:
There's a much earlier "Let _httpCache_ be null." step, no?

Reply:
You're correct - completely missed it, given how far up it is.